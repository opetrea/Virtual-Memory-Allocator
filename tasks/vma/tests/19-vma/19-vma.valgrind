==91899== Memcheck, a memory error detector
==91899== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==91899== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==91899== Command: /home/student/Desktop/tema1_sda/sd-2023-homeworks/vma
==91899== Parent PID: 91898
==91899== 
==91899== 
==91899== HEAP SUMMARY:
==91899==     in use at exit: 0 bytes in 0 blocks
==91899==   total heap usage: 41 allocs, 41 frees, 9,240 bytes allocated
==91899== 
==91899== All heap blocks were freed -- no leaks are possible
==91899== 
==91899== For lists of detected and suppressed errors, rerun with: -s
==91899== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
