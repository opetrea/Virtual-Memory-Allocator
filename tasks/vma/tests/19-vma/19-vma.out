Total memory: 0x14 bytes
Free memory: 0x10 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 4

Block 1 begin
Zone: 0x3 - 0x7
Miniblock 1:		0x3		-		0x4		| RW-
Miniblock 2:		0x4		-		0x5		| RW-
Miniblock 3:		0x5		-		0x6		| RW-
Miniblock 4:		0x6		-		0x7		| RW-
Block 1 end
Total memory: 0x14 bytes
Free memory: 0x11 bytes
Number of allocated blocks: 2
Number of allocated miniblocks: 3

Block 1 begin
Zone: 0x3 - 0x4
Miniblock 1:		0x3		-		0x4		| RW-
Block 1 end

Block 2 begin
Zone: 0x5 - 0x7
Miniblock 1:		0x5		-		0x6		| RW-
Miniblock 2:		0x6		-		0x7		| RW-
Block 2 end
Total memory: 0x14 bytes
Free memory: 0x10 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 4

Block 1 begin
Zone: 0x3 - 0x7
Miniblock 1:		0x3		-		0x4		| RW-
Miniblock 2:		0x4		-		0x5		| RW-
Miniblock 3:		0x5		-		0x6		| RW-
Miniblock 4:		0x6		-		0x7		| RW-
Block 1 end
Total memory: 0x14 bytes
Free memory: 0x11 bytes
Number of allocated blocks: 2
Number of allocated miniblocks: 3

Block 1 begin
Zone: 0x3 - 0x5
Miniblock 1:		0x3		-		0x4		| RW-
Miniblock 2:		0x4		-		0x5		| RW-
Block 1 end

Block 2 begin
Zone: 0x6 - 0x7
Miniblock 1:		0x6		-		0x7		| RW-
Block 2 end
